src/adapters/rubicon.js

```diff
@@ -152,22 +224,32 @@ function RubiconAdapter() {
 
   function handleRpCB(responseText, bidRequest) {
     let responseObj = JSON.parse(responseText); // can throw
+    var ads = responseObj.ads;
+
+    console.log('bid request', bidRequest);
+    console.log('response', responseObj);
 
-    if(
-      typeof responseObj !== 'object' ||
-      responseObj.status !== 'ok' ||
-      !Array.isArray(responseObj.ads) ||
-      responseObj.ads.length < 1
-    ) {
```

Comment:
I believe these checks are identical for both display and video. Only difference was grabbing the video ads response correctly from frankenstein which comes back as an objet and not an array directly because it implements SRA. If further individual validation becomes necessary in the future, we can break it out based on mediaType.